tasks/install.yml

```diff
@@ -71,4 +71,4 @@
     path: "{{ redis_install_dir }}/bin/{{ item }}"
     link: "/usr/bin/{{ item }}"
   with_items: "{{ redis_binaries.stdout_lines }}"
-  when: redis_binaries is succeeded
+  when: not ansible_check_mode and redis_binaries is succeeded
```

Comment:
rather than this can we run `check_mode: no` on the list command as that isn't ever going to modify the file system